static/manifest.json

```diff
@@ -4,9 +4,8 @@
   "description": "Learn AMP by Examples",
   "start_url": "https://ampbyexample.com",
   "display": "standalone",
```

Comment:
As long as the device has a back button everything works fine (even navigating to Github). I'd leave it for know for demo purposes, but we definitely have to keep this in mind. For example, we would need to add a sharing button in standalone mode as there is no other way to access the URL. 

BTW which devices don't have a back button (besides Amazon Fire with manually installed Chrome)?